python-sgd-regression/sgd_regression.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python3
+
+# QUESTION: when creating files in python-sgd-regression I copied folder python-linear-regression and just renamed
+# `linear-regression` to `sgd-regression`. Is there another way? e.g. cookiecutter template or something?
+
```

Comment:
A template would be awesome, but there are many variants to manage (Java, Python, Python + Scikit...) and we are still changing things. I'm starting to use Atomist.com to organise the projects, but creating templates is a hassle for far.

Reply:
Agree. Didn't know about atomist, looks neat.